test/Npgsql.Tests/DistributedTransactionTests.cs

```diff
@@ -365,12 +350,7 @@ public void Connection_reuse_race_chaining_transaction([Values(false, true)] boo
             }
             catch (Exception ex)
             {
-                Assert.Fail(
-                    @"Failed at iteration {0}.
-Events:
-{1}
-Exception {2}",
-                    i, FormatEventQueue(eventQueue), ex);
+                Assert.Fail($"Failed at iteration {i}.\r\nEvents:\r\n{FormatEventQueue(eventQueue)}\r\nException {ex}");
```

Comment:
This doesn't look like a good change to me (especially since it encodes a particular type of newline sequence). I'd convert this to a raw literal string instead (`"""`).